examples/bri-3/.env.sample

```diff
@@ -19,3 +19,4 @@ BPI_ENCRYPTION_KEY_KTY_PARAM="oct"
 SNARKJS_CIRCUITS_PATH="zeroKnowledgeArtifacts/circuits/"
 VSM_CYCLE_PERIOD_IN_SECS=15
 VSM_CYCLE_TX_BATCH_SIZE=5
+HASH_ALGORITHM="sha256"
```

Comment:
Please also add a short explanation of the param to the readme file, where env vars are documented.